_data/infrastructure-projects/police-stations-hash147.yaml

```
@@ -0,0 +1,34 @@
+name: Police Stations
+coordinates:
+  - -30.4759, 30.2030
+stage: Various
+total_budget: 142200000000
+department: Police
+description: Construction of new and re-established police stations
+projected_budget: 51800000000
+department_link: /path/to/department/detail/page
+nature_of_investment: Upgrading and additions
+infrastructure_type: Police stations
+google_maps_link: https://www.google.com/maps/@-30.4759,30.2030,17z
+data:
+  - year: 2015-16
+    type: actual
```

I'm checking whether these labels should be... will confirm.
2015/16  Audit outcome
2016/17  Audit outcome
2017/18  Audit outcome
2018/19  Adjusted appropriation
2019/20  MTEF
2020/21  MTEF
2021/22  MTEF

---

@otterbotter @schalkventer this is correct - these "types" should be budget phase with the above values (excluding the year)
```suggestion
    budget_phase: Audit outcome
```